CONTRIBUTING.md

```diff
@@ -0,0 +1,208 @@
+# Contributing
+
+This document provides guidelines for contributing to this repository.
+
+## Development Environment Setup
+
+### Prerequisites
+* Basic tools: `curl`, `jq`, `oc`
+* go: v1.19
```

Comment:
Go tests are written with v1.19 and some functions might not be compatible with higher versions while running the tests.

Reply:
its also what `go.mod` says.

Reply:
Since having go installed is a requirement for parsing go.mod (In other words, unlike with `pipenv` we don't have a tool to automatically get the right Go version on demand according to a file in the repo), we should specify it here so users will know what to install.

And yes, a basic requirements doc typically needs to get updates when you change the requirements...